traversal/select_links_test.go

```diff
@@ -1,25 +1,30 @@
 package traversal_test
 
 import (
+	"reflect"
 	"testing"
 
-	. "github.com/warpfork/go-wish"
+	qt "github.com/frankban/quicktest"
+	"github.com/google/go-cmp/cmp"
 
 	"github.com/ipld/go-ipld-prime/datamodel"
 	"github.com/ipld/go-ipld-prime/traversal"
 )
 
 func TestSelectLinks(t *testing.T) {
+
+	linkEquals := qt.CmpEquals(cmp.Exporter(func(reflect.Type) bool { return true }))
```

Comment:
document why this is needed? I would assume that qt.DeepEquals would be equivalent to ShouldEqual

Reply:
Fat fingers on my part; this should not be there. thank you for pointing that out 🍻